src/ray/raylet/main.cc

```diff
@@ -5,6 +5,28 @@
 #include "ray/stats/stats.h"
 #include "ray/status.h"
 
+#include "gflags/gflags.h"
+
+DEFINE_string(raylet_socket_name, "", "The socket name of raylet.");
+DEFINE_string(store_socket_name, "", "The socket name of object store.");
+DEFINE_int32(object_manager_port, -1, "The port of object manager.");
+DEFINE_int32(node_manager_port, -1, "The port of node manager.");
+DEFINE_string(node_ip_address, "", "The ip address of this node.");
+DEFINE_string(redis_address, "", "The ip address of redis server.");
+DEFINE_int32(redis_port, -1, "The port of redis server.");
+DEFINE_int32(num_initial_workers, 0, "Number of initial workers.");
+DEFINE_int32(maximum_startup_concurrency, 1, "Maximum startup concurrency");
+DEFINE_string(static_resource_list, "", "The static resource list of this node.");
+DEFINE_string(config_list, "", "The raylet config list of this node.");
+DEFINE_string(python_worker_command, "", "Python worker command.");
+DEFINE_string(java_worker_command, "", "Java worker command.");
+DEFINE_string(redis_password, "", "The password of redis.");
+DEFINE_string(temp_dir, "", "Temporary directory.");
+DEFINE_bool(disable_stats, false, "Whether disable the stats.");
+DEFINE_string(stat_address, "127.0.0.1:8888", "The address that we report metrics to.");
+DEFINE_bool(disable_stdout_exporter, true,
+            "Whether disable the stdout exporter for stats.");
```

Comment:
maybe change this to `enable_stdout_exporter` with default value false. Then we can just use `--enable_std_exporter`, instead of `--disable_stdout_exporter=false`.